velox/dwio/parquet/reader/ParquetData.cpp

```diff
@@ -143,4 +212,42 @@ std::pair<int64_t, int64_t> ParquetData::getRowGroupRegion(
   return {fileOffset, length};
 }
 
+std::unique_ptr<BloomFilter> getBloomFilter(
+    dwio::common::BufferedInput& bufferedInput,
+    const std::vector<thrift::RowGroup>& rowGroups,
+    const uint32_t rowGroupId,
+    const uint32_t column,
+    memory::MemoryPool& pool) {
+  if (rowGroupId < 0 || rowGroupId >= rowGroups.size()) {
+    VELOX_FAIL("Invalid row group ordinal: ", rowGroupId);
+  }
+  auto rowGroupMetadata = rowGroups[rowGroupId];
+  if (column < 0 || column >= rowGroupMetadata.columns.size()) {
+    VELOX_FAIL("Invalid column index at column ordinal ", column);
+  }
+
+  auto colChunk = rowGroupMetadata.columns[column];
+  if (colChunk.__isset.crypto_metadata) {
+    VELOX_NYI("Cannot read encrypted bloom filter yet");
+  }
+
+  if (!colChunk.meta_data.__isset.bloom_filter_offset) {
```

Comment:
I would like to keep it here so for any tests that we write, we get a null bloom filter rather than trying to read a non-existent bloom filter. Will change the call site.

Reply:
Will retain this check at the call site too, so we can assume that bloom filter should always be present if the offset is set.